fuzz/src/lib.rs

```diff
@@ -0,0 +1,134 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
```

Comment:
The functions in this file seem to be copied from the neqo-transport crate.  Maybe you can just move them to the test fixture instead.

Reply:
See #1830. When that's merged, I'll rebase this PR.